src/Assign/README.md

```diff
@@ -0,0 +1,140 @@
+# Assign
+```haskell
+Assign Object
+```
+`Assign` is a `Monoid` that will combine two objects using `Object.assign` on
+their `Object` values, merging and overwriting new values for values that are
```

Comment:
> on their `Object` values,

May be over kill, we should 🔪 that and just and go with:

using `Object.assign` to merge the (2) `Object`s.

My reasoning is that by this point people should know what `Object.assign` is, and if not it gives them a nice little treasure hunt for knowledge.

Reply:
Agreed!